README.md

```diff
@@ -79,5 +79,5 @@ terminology, etc. is wellcome.
 
 For more details see the wiki pages and the manual page:
  - https://github.com/dashohoxha/egpg/wiki
- - http://dashohoxha.github.io/egpg/gnupg-2.1/man/
+ - http://dashohoxha.fs.al/easy-gnupg-presentation/
```

Comment:
The correct link is: https://easygnupg.github.io/egpg/gnupg-2.1/man/

Reply:
Also the link for the /gnupg-2.0/man/ should be fixed.

Reply:
Basically, the domain `dashohoxha.github.io` should be replaced by `easygnupg.github.io` everywhere. It needs a find/replace to all files. I don't know which editor you are using, but in Emacs this is an easy task.

Reply:
Yes, using sed too.